publish.js

```diff
@@ -323,16 +323,26 @@ function buildMemberNav(items, itemHeading, itemsSeen, linktoFn) {
             var members = find({kind:'member', memberof: item.longname});
             var docdash = env && env.conf && env.conf.docdash || {};
             var conf = env && env.conf || {};
+            var classes = '';
+
+            // show private class?
+            if (docdash.private === false && item.access === 'private') return;
+
+            if (docdash.navLevel >= 0 && item.ancestors.length > docdash.navLevel) {
```

Comment:
Two things just to note here
1. While `undefined >= 0` is false, I would not want to rely on such language/environment dependent assumptions which are not so obvious at first look. (specifically conversion/comparison between numbers and none numbers)So maybe we could check if navLevel exist at all and provide backwards compatibility that way, for configs where it is not defined

2. How should it play together with collapsible menu option? If menu is collapsed by `collapse` setting, does it still make sense to hide nth level items? I am not saying it does not make sense, I just want to have your opinion :)

Reply:
The collapse option is orthogonal to level -- it applies to class members (props, methods), whereas level applies to sub-classes.